Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Aspinwall/Aspinwall Shell](https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/).


It also includes following components:

* [Aspinwall/Widgets: Welcome](https://hosted.weblate.org/projects/aspinwall-ui/widget-welcome/)

* [Aspinwall/Widgets: Notepad](https://hosted.weblate.org/projects/aspinwall-ui/widget-notepad/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/aspinwall-ui/-/aspinwall-shell/horizontal-auto.svg)

comradekingu committed:
Added translation using Weblate (Norwegian Bokmål)

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 83.3% (5 of 6 strings)

Translation: Aspinwall/Widgets: Welcome
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/widget-welcome/nb_NO/

comradekingu committed:
Added translation using Weblate (Norwegian Bokmål)

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (3 of 3 strings)

Translation: Aspinwall/Widgets: Notepad
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/widget-notepad/nb_NO/

comradekingu committed:
Added translation using Weblate (Norwegian Bokmål)

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 70.5% (55 of 78 strings)

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/nb_NO/

knuxify commented:
Holding off from merging these until mid-April or late April.

(EDIT: nevermind, i have to push this to mid-May sadly.)

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 92.3% (72 of 78 strings)

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/nb_NO/

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 93.5% (73 of 78 strings)

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/nb_NO/

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (6 of 6 strings)

Translation: Aspinwall/Widgets: Welcome
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/widget-welcome/nb_NO/

weblate committed:
Update translation files

Updated by "Update LINGUAS file" hook in Weblate.

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/

weblate committed:
Update translation files

Updated by "Update LINGUAS file" hook in Weblate.

Translation: Aspinwall/Widgets: Notepad
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/widget-notepad/

weblate committed:
Update translation files

Updated by "Update LINGUAS file" hook in Weblate.

Translation: Aspinwall/Widgets: Welcome
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/widget-welcome/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: Aspinwall/Aspinwall Shell
Translate-URL: https://hosted.weblate.org/projects/aspinwall-ui/aspinwall-shell/
